### PR TITLE
[WIP] Shuttle coordinate input

### PR DESCRIPTION
**Pull request by unid15**
Adds a shuttle disk with a special driver which allows converting coordinates into destination points

image

This is extremely unfinished and unpolished but I'll still put it up for general discussion/feedback/ideas

Disk can't be obtained currently. It doesn't make the shuttle able to destroy station areas (though that might be a cool traitor item idea)

### CODE REVIEW

**Comment by strawberrytan** [comment]
Maybe don't let it dock on any blueprint defined rooms

**Comment by Blithering** [comment]
this could be useful for the syndicate elite and deathsquad shuttles, as they tend to be called when shits gone really down the pan and the usual shuttle docks are either in the middle of a blob mass or floating debris

**Comment by MadmanMartian** [comment]
There doesn't look to be anything wrong with this

File: code/game/machinery/computer/shuttle_computers.dm [hunk 1]
```diff
@@ -362,6 +385,39 @@
...
+			shuttle.linked_port.x + custom_x,\
+			shuttle.linked_port.y + custom_y,\
+			shuttle.linked_port.z + custom_z
+			)
+
+			if(!dest)
+				to_chat(usr, "Error! Bad coordinates.")
+				return
+
+			disk.destination = new /obj/docking_port/destination(dest)
```
Comment (DamianX) [inline code/game/machinery/computer/shuttle_computers.dm:413]: Is this ever deleted or does it keep creating a new docking_port each time it travels?

**Pull request closed without merging**